modules/ROOT/pages/6.4-release-notes.adoc

```diff
@@ -0,0 +1,1154 @@
+
+For information on the **Advanced Template** plugin, see: xref:advanced-template.adoc[Advanced Templates].
+
+
+[[accompanying-premium-plugin-changes]]
+== Accompanying Premium Plugin changes
+
+The following premium plugin updates were released alongside {productname} 6.4.
+
+=== Advanced Typography 1.1.1
+
+The {productname} 6.4 release includes an accompanying release of the **Advanced Typography** premium plugin.
+
+**Advanced Typography** 1.1.1 includes the following fixes, additions, and improvements.
+
+=== Fixed adding redundant space between a comma and a quote by switching the sequence of executing safe and unsafe rules
+// TINY-9442
+
+After {productname} introduced the Smart Typography plugin, a bug was discovered that was causing a space to be introduced between a comma and a closing double-quote mark, as shown in the example below:
+
+.Example
+> “They went that way,[space introduced here]” he said as he pointed.
+
+As demonstrated in the example above, an unintentional or redundant space was inserted into a separate node position.
+
+{productname} fixed the issue by changing the sequence of executing safe and unsafe rules surrounding Typography.
+
+In {productname} version 6.4, the Smart Typography plugin no longer creates a space between a comma and a closing double-quote mark.
+
+=== Added common/space/delBetweenExclamationMarks to remove spaces between exclamation marks
+// TINY-9398		
+
+An issue was discovered that affected the typography plugin in previous versions of {productname} when inserting content into the editor. Specifically, the issue involved exclamation marks and the spaces between them, such as in the example `“Hello! !“`. This issue prevented the editor from removing the spaces between the double *exclamation marks* `! !` after executing the typography plugin toolbar or menubar button.
+
+The fix for this involved adding a new rule which removes the spaces between exclamation marks when the typography button is pressed.
+
+In {productname} 6.4, spaces are now removed between exclamation marks `!!` for formatted HTML documents when the typography plugin is used. To learn more about Advanced Typography and how to add this new premium plugin to your project, please refer xref:advanced-typography.adoc[Advanced Typography].
+
+=== Added typography plugin to the default menubar
+// TINY-9450		
+
+In previous versions of {productname}, the typography button would *not* appear in the `menubar` by default. As a consequence, the user could only `find` the typography button in the `toolbar` due to it not appearing in the `menubar` dropdown.
+
+To correct this bug, the typography plugin was added to the default `menubar` config.
+
+{productname} 6.4, by default, will now display the typography button in both the `toolbar` and below the “Source code" button in the `menubar`.
```

Comment:
```suggestion
In the previous version of {productname}, the typography menu item was *not* available in the `**Tools**` menu from the `menubar` by default.

In {productname} 6.4, the typography menu item will display in the `**Tools**` menu by default, given that the `typography_default_lang` option is set.
```
Typography was only available in 6.3, so the previous version. The typography button was only visible from the toolbar if it was added to the toolbar, not by default. The menu item was added as a default to the tools menu in the menubar. In 6.4 the typography button will still not display in the toolbar by default. In 6.4 the typography menu item will only display by default in the tools menu from the menu bar if the `typography_default_lang` option is set, as it is required for the plugin.